logos/src/logos/main.py

```
@@ -3591,6 +3592,29 @@ async def add_provider(data: AddProviderRequest):
     await refresh_pipeline_runtime_state()
     return result
 
+@app.post("/logosdb/update_provider", tags=["admin"])
+async def update_provider(data: UpdateProviderRequest):
```

Flake8 (7.3.0)

[error] 'UpdateProviderRequest' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)